wordpress-mobile/AztecEditor-iOS
# Rendering Figcaptions!

### Description:
In this PR we're adding `figcaption` rendering capabilities.

Needs Review: @diegoreymendez 

Closes #723

### Details:
- **ImageAttachment** will, now, also serialize the `caption` field (NSCoder implementation).
- Updated **MediaAttachment** / **ImageAttachment** bounds calculation methods
- New **MediaAttachment.Appearance** property
- And of course, the actual Caption rendering mechanisms!


### Alignment / Sizing:
Our goal is for the figcaption to render as follows:
<img width="878" alt="screen shot 2017-12-21 at 13 26 00" src="https://user-images.githubusercontent.com/1195260/34267400-6da7dbca-e65b-11e7-9d1b-bcedc68b226c.png">


--

### Testing:
1. Launch the Demo App
2. Open the `Captions Demo`

Verify that all of the captions look as expected!
